src/signs/ui/display/SignDisplay.tsx

```diff
@@ -1,38 +1,50 @@
 import React from 'react'
-import { Container } from 'react-bootstrap'
+import { Col, Container, Row } from 'react-bootstrap'
 import { RouteComponentProps } from 'react-router-dom'
 import withData, { WithoutData } from 'http/withData'
-import SignsService from 'signs/application/SignsService'
+import SignService from 'signs/application/SignService'
 import Sign from 'signs/domain/Sign'
 import './signDisplay.css'
 import WordService from 'dictionary/application/WordService'
 import SignInformation from 'signs/ui/display/SignInformation'
 import SignHeading from 'signs/ui/display/SignHeading'
-import MesZl from 'signs/ui/display/MesZl'
 import AppContent from 'common/AppContent'
 import { SectionCrumb } from 'common/Breadcrumbs'
 import SessionContext from 'auth/SessionContext'
 import { Session } from 'auth/Session'
+import MesZlContent from 'signs/ui/search/MesZLContent'
 
 function SignDisplay({
   sign,
   wordService,
+  signService,
 }: {
   sign: Sign
+  signService: SignService
   wordService: WordService
 }): JSX.Element {
   return (
-    <AppContent crumbs={[new SectionCrumb('Signs')]}>
+    <AppContent
+      crumbs={[
+        new SectionCrumb('Signs'),
+        new SectionCrumb(sign.displaySignName),
```

Comment:
The sign is not a section and the crumb for the current page should not be a link. Use `TextCrumb`.